.github/workflows/ci.yml

```diff
@@ -25,6 +25,8 @@ on:
   pull_request:
     branches:
       - main
+  schedule:
+    - cron: "0 20 * * *"
```

Comment:
Nightly releases every day at 2:00 UTC+8, so this might be better running 4:00 UTC+8 (`0 12 * * *`)

Reply:
Ahhhh yes. My bad math. Rebase and we'll be good to go